<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeriodicTableWidget</class>
 <widget class="QWidget" name="PeriodicTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="pteList"/>
   </item>
   <item>
    <widget class="QWidget" name="pteContainer" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="0">
       <widget class="QLabel" name="colLabel">
        <property name="text">
         <string>Display color:</string>
        </property>
        <property name="buddy">
         <cstring>colSel</cstring>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="3">
       <widget class="QPushButton" name="defaultBut">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Reset to Element defaults</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QLineEdit" name="pwppSel"/>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QLineEdit" name="cpppSel"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cutLabel">
        <property name="text">
         <string>Bo&amp;nd cutoff radius:</string>
        </property>
        <property name="buddy">
         <cstring>cutSel</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="vdwSel">
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="3">
       <widget class="QPushButton" name="fromGlobalBut">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load from global table</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QSpinBox" name="zSel">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="mSel">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="cutSel">
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="zLabel">
        <property name="text">
         <string>Ato&amp;mic number:</string>
        </property>
        <property name="buddy">
         <cstring>zSel</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="2">
       <widget class="QLineEdit" name="cpnlSel"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="mLabel">
        <property name="text">
         <string>Mass:</string>
        </property>
        <property name="buddy">
         <cstring>mSel</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="vdwLabel">
        <property name="text">
         <string>Van-der-Waals radius:</string>
        </property>
        <property name="buddy">
         <cstring>vdwSel</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="pwppLabel">
        <property name="text">
         <string>PWScf PP-File:</string>
        </property>
        <property name="buddy">
         <cstring>pwppSel</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QPushButton" name="colSel">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="covSel">
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="cpppLabel">
        <property name="text">
         <string>CPMD PP-File:</string>
        </property>
        <property name="buddy">
         <cstring>cpppSel</cstring>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="cpnlLabel">
        <property name="text">
         <string>CPMD Nonlocality:</string>
        </property>
        <property name="buddy">
         <cstring>cpnlSel</cstring>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="3">
       <widget class="QPushButton" name="toGlobalBut">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save to global table</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="helpButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="covLabel">
        <property name="text">
         <string>Co&amp;valent radius:</string>
        </property>
        <property name="buddy">
         <cstring>covSel</cstring>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="3">
       <widget class="QPushButton" name="deleteBut">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete custom Element</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pteList</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>PeriodicTableWidget</receiver>
   <slot>setEntry(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toGlobalBut</sender>
   <signal>clicked()</signal>
   <receiver>PeriodicTableWidget</receiver>
   <slot>changeEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromGlobalBut</sender>
   <signal>clicked()</signal>
   <receiver>PeriodicTableWidget</receiver>
   <slot>changeEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultBut</sender>
   <signal>clicked()</signal>
   <receiver>PeriodicTableWidget</receiver>
   <slot>changeEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteBut</sender>
   <signal>clicked()</signal>
   <receiver>PeriodicTableWidget</receiver>
   <slot>changeEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setEntry(QListWidgetItem*)</slot>
  <slot>changeEntry()</slot>
 </slots>
</ui>
